### ISSUE TITLE
define a generic labeling scheme for the PNG images, for labeling LCO and LSST images

### ISSUE THREAD

**Issue by srodney** [srodney/diffimageml#114]
Each PNG needs to be labeled with a particular class label, which the CNNs will train on.   We want this to be extensible, allowing for the possibility of adding new classes in the future. 

Probably we want a class definition for this, and it should follow an existing schema that works with, say, the TensorFlow Keras preprocessing library.

See

 https://www.tensorflow.org/api_docs/python/tf/keras/preprocessing/image_dataset_from_directory

**Comment by srodney** [comment]
After reading and thinking on this, let's keep it simple: 
* the class labels will be defined by directory names.  So our three-class schema would be :
  *  notreal ,  single,  multiple
* each PNG that is input to the CNN will contain info from a triplet of images. So I'll suggest that we store these as RGB values:
  * R=template ;  G=search ;  B=diff
 
 Using the keras preprocessing image_dataset_from_directory should then be sufficient to capture all of the info cleanly into separate classes, without needing a new Python object Class defined within diffimageML.
